src/datatypes/src/schema/column_schema.rs

```diff
@@ -28,6 +28,7 @@ pub type Metadata = HashMap<String, String>;
 
 /// Key used to store whether the column is time index in arrow field's metadata.
 pub const TIME_INDEX_KEY: &str = "greptime:time_index";
+pub const COMMENT_KEY: &str = "greptime:storage:comment";
```

Comment:
If the comment key is in datatypes crate, I think we should change the key to `greptime:comment`.

But this may cause breaking change.

Reply:
Yes, that's the problem, so I leave it the same as before.